modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -0,0 +1,76 @@
+
+[console, gsql]
+---
+$ gadmin backup list
+|================================================================================================================================================
+|             TAG              |    TYPE     | VERSION |  SIZE  |             BASE             | POINT-IN-TIME START TIME |      CREATED AT      
+| backup-2024-06-13T061939.131 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T06:19:39Z     | 2024-06-13T06:19:39Z 
+| backup-2024-06-13T063915.054 | INCREMENTAL | 4.1.0   | 1.5 MB | backup-2024-06-13T061939.131 | 2024-06-13T06:19:39Z     | 2024-06-13T06:39:15Z 
+| backup-2024-06-13T081939.120 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T08:19:39Z     | 2024-06-13T08:19:39Z 
+|================================================================================================================================================
+---
+
+here, the time range for backup `backup-2024-06-13T061939.131` is `[2024-06-13T06:19:39Z,2024-06-13T06:19:39Z]`, therefore, for a full backup is just a point. As for the incremental backup `backup-2024-06-13T063915.054`, its range is `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]`. so, for instance, if we have performed a restore at `2024-06-14T06:19:39Z`. And then we can’t restore to the timepoint `2024-06-13T07:00:00Z`, for it is not in any backup's time range. however, `2024-06-13T06:35:00Z` is supported for it is covered by the time range `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]` of the backup `backup-2024-06-13T063915.054`.
+
+
+[NOTE]
+====
+because the restore action will make the uncovered timepoint unavailable to restore, if you are not very sure about the time point to restore,  we’d better ensure the possible time points are all covered by existing backups. if there is not this kind of backup, you may create a by `gadmin backup create <backup_tag> --incremental --base <full_backup_tag>`, the new backup will cover the range from the full backup <full_backup_tag>'s create time to the current time.
+====
+
+== Key Benefits
+
+The primary advantage of Point-in-Time Restore is the flexibility it offers in selecting any specific time point on the timeline for restoration.
+
+.Key Benefit Scenario
+====
+. A user creates a full backup on __Sunday__.
+
+. During the week, user creates a differential backup on every day 10pm.
+
+. On Thursday, user finds there are unwanted changes made on Wednesday around 2pm.
+
+. User can restore system to 1:55pm of Wednesday instead of the backup made on Tuesday 10pm.
+
+. The changes made between Tuesday 10pm and Wednesday 1:55pm are saved this way.
+====
+
+Summery of key benefits:
+
+* Precise recover time point to save a much as possible correct work made to database.
+
+
+== Limitations and Considerations
+
+Currently, the Point-in-Time Restore feature has these known limitations:
+
+.Point-in-Time Restore examples.
+image::PIT_restore.png[]
+
```

Comment:
Let's have a sub-heading for each limitations:
1. GSQL Content: The GSQL content, such as schemas and RBAC settings, cannot be restored to the exact version at the provided time point. After a Point-in-Time Restore, these elements will reflect the version of the nearest differential backup used.
2. Precision Window: Point-in-Time Restore operates with a precision window of a few minutes. To ensure optimal data consistency, it is recommended to restore to a time point at least 2-3 minutes before the timestamp of any unwanted changes.
3. Coverage: Point-in-Time Restore cannot be performed for timestamps not covered by an existing differential backup. If the desired time point is not covered, a new differential backup must be created before performing the restore."

Reply:
Done